tests/main/snap-env/task.yaml

```diff
@@ -10,7 +10,8 @@ debug: |
 execute: |
     echo "Collect SNAP and XDG environment variables"
     test-snapd-tools.env | egrep '^SNAP_' | egrep -v '^SNAP_DID_REEXEC'| sort > snap-vars.txt
-    test-snapd-tools.env | egrep '^XDG_' | sort > xdg-vars.txt 
+    test-snapd-tools.env | egrep '^XDG_' | sort > xdg-vars.txt
+    test-snapd-tools.env | egrep '^EXTRA_' | sort > extra-vars.txt 
```

Comment:
I have a some extra tests in #2765 but I'll gladly de-conflict whatever lands second.